### PR TITLE
[Symfony4] Skip ConsoleExecuteReturnIntRector on ternary returns number

### LINKED ISSUE #5733: Incorrect behavior of ConsoleExecuteReturnIntRector
# Bug Report

| Subject        | Details              |
| :------------- | :--------------------|
| Rector version | 0.9.31      |
| Installed as   | composer dependency  |

## Minimal PHP Code Causing Issue

See https://getrector.org/demo/d163eaf1-f599-4f4a-9683-5d593dd40b79

```php
<?php

function returns_bool(): bool {
    return true;
}

class RunCommand extends \Symfony\Component\Console\Command\Command
{
    protected function execute(\Symfony\Component\Console\Input\InputInterface $input, \Symfony\Component\Console\Output\OutputInterface $output): int
    {
        $boolVariable = returns_bool();
        
        return $boolVariable ? 0 : 1;
    }
}
```

### Responsible rules

* `ConsoleExecuteReturnIntRector`

## Expected Behavior

The results of ternary expression `$boolVariable ? 0 : 1` is always an integer, so there is no need to typecast the result as suggested by Rector:
```diff
-        return $boolVariable ? 0 : 1;
+        return (int) ($boolVariable ? 0 : 1);
```

BTW this works properly when `$boolVariable` value is directly associated like this: https://getrector.org/demo/15150d54-496d-4f42-a179-5e157c67bfc0

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes #5733

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/skip_ternary_return_int.php.inc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class SkipTernaryReturnInt extends Command
+{
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        return rand(0, 1) ? 0 : 1;
+    }
```
Comment (TomasVotruba) [inline rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/skip_ternary_return_int.php.inc:14]: What about this case?

```php
$yes = 0;
$no = 1;
return rand(0, 1) ? $yes : $no;
```

Comment (samsonasik) [reply rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/skip_ternary_return_int.php.inc:14]: implemented https://github.com/rectorphp/rector/pull/5735/commits/6be8c8fd525ebb5c4a058dca9b558a1f63be4dd8

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Looks good, thank you :+1:

**Comment by OndraM** [comment]
Nice jobs, thanks!

**Pull request merged**